components/textbook-demo/PrerequisiteMaterialSection.vue

```diff
@@ -51,15 +51,15 @@ export default class PrerequisiteMaterialSection extends Vue {
         action: `${this.segmentActionBase} python-and-jupyter-notebooks`
       },
       url:
-        'https://platypus.qiskit.org/course/ch-prerequisites/introduction-to-python-and-jupyter-notebooks'
+        'https://learn.qiskit.org/course/ch-prerequisites/introduction-to-python-and-jupyter-notebooks'
     },
     {
       title: 'What is Quantum?',
       description:
         '‘Quantum physics’ is a term widely used but much less understood. It is a mathematical model first used to describe the...',
       segment: { action: `${this.segmentActionBase} what-is-quantum` },
       url:
-        'https://platypus.qiskit.org/course/getting-started/what-is-quantum'
+        'https://learn.qiskit.org/course/getting-started/what-is-quantum'
```

Comment:
This URL leads to a 404 page.

It might not be an issue with this PR but rather a missing update on the Platypus project, but I wanted still to point that out just in case.

Reply:
This link has been remove in [this PR](https://github.com/Qiskit/qiskit.org/pull/1900) because now it's part of the course and we don't need it anymore on the prerequisites section :)